package.json

```diff
@@ -28,7 +31,8 @@
     "test:server:watch": "gulp test:server:watch",
     "migrate": "migrate",
     "update": "npm install --quiet && npm prune && migrate",
-    "postinstall": "bower install --allow-root && bower prune --allow-root"
+    "postinstall": "bower install --allow-root && bower prune --allow-root",
+    "agendash": "agendash --db=mongodb://localhost/trustroots-dev --collection=agendaJobs --port=3001"
```

Comment:
Agendash could be mentioned at [Install instructions](https://github.com/Trustroots/trustroots/blob/master/INSTALL.md#running--development)?